internal/outputs/console_collector.go

```diff
@@ -41,7 +43,9 @@ func (p *consoleResultCollector) Close() error {
 	p.out.Printf("\n")
 	p.out.InitTabs(95)
 	for _, policy := range report.Policies {
-		p.out.ColorPrintf("%s [bold][light_gray][%s][yellow] %s[reset]: %s\n", ICON_MAGNIFIER, policy.PolicyGroup, policy.PolicyName, policy.PolicyTitle)
+		policyTitleWithLink := fmt.Sprintf("%s \x1b]8;;%s\x07%s\x1b]8;;\x07", ICON_HYPERLINK, policy.ExternalURI, policy.PolicyTitle)
```

Comment:
Would it be too verbose to display the recommendation as well?
Maybe just for those with violations?

Reply:
The recommendation will be displayed only in JSON or Security Command Center output.

For console output, there will be a terminal link only:
![image](https://user-images.githubusercontent.com/20063540/194081015-fd980d2f-f21c-48dd-b9e0-639eae1b4612.png)

Visually it will look almost same as now, what changed:
* The "link" icon was added before the policy title
* The policy title will be "clickable" link now (URL won't be displayed)